models/service/intermediate/int_hubspot__ticket_calendar_spine.sql

```diff
@@ -9,13 +9,13 @@
         file_format = 'parquet'
     )
 }}
-
+-- depends_on: {{ ref('stg_hubspot__ticket') }}
```

Comment:
I worry this will now cause compilation errors for customers who try to run `dbt compile` when first trying to use the package.

What is the limitation of ensuring this model still references the source?

Reply:
i'll add the same check i just added to zendesk so that non-unioning users will still use the `source`